templates/sample/src/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Blockly Demo: Generating Javascript</title>
+    <script src="../js/blockly_compressed.js"></script>
+    <script src="../js/javascript_compressed.js"></script>
+    <script src="../js/blocks_compressed.js"></script>
+    <script src="../msg/js/en.js"></script>
+
+    <script src="./csblocks.js"></script>
+    <link rel="stylesheet" href="csblocks.css">
+
+    <style>
+    </style>
+</head>
+
+<body>
+    <p>
+        <button onclick="runGame()">Run Block Game</button>
+    </p>
+    <div id="blocklyDiv" style="height: 480px; width: 600px;"></div>
+    <xml id="toolbox" style="display: none;">
+        <category name="Logic">
+            <block type="lightswitch">
+                <field name="lightcolor">red</field>
+                <field name="switch">on</field>
+            </block>
+        </category>
+    </xml>
+
+    <div id="square"></div>
+
+    <script>
```

Comment:
this script should be defined in `static/js` and imported here as a bundle (see `templates/front/game.html` how the main bundle is imported). That way you can import Blockly using the npm blockly package instead of downloading and linking the blockly compressed js file.

Reply:
I changed the name of file from index.html to sample.html. After that I tested and it works find and we can only keep html files under template folder. Hopefully it satisfy the requirement of Django structure.